app/Models/GamesCoupdecoeur.php

```diff
@@ -34,11 +34,11 @@ class GamesCoupdecoeur extends Model
 
     protected $guarded = [];
 
-    public function game(){
+    public function game() {
         return $this->hasOne('App\Models\Game', 'id', 'game_id')->with('maker', 'developers', 'gamefiles');
     }
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->